tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py

```diff
@@ -982,6 +986,16 @@ def _MakeSavedModel(self, run_params):
     return self._MakeSavedModelV1(run_params)
 
   def RunTest(self, run_params):
+    tf32_is_allowed = config.tensor_float_32_execution_enabled()
+    try:
+      config.enable_tensor_float_32_execution(self.ShouldAllowTF32Computation())
+
+      logging.info("TensorFloat32 Arithmetic Enabled: {}".format(
+        config.tensor_float_32_execution_enabled()
+      ))
+    except:
+      pass
```

Comment:
I have two questions for this code block:
(1) why do we need to save the value and restore it in line 1065?
(2) why do we need to put config.enable_tensor_float_32_xxx to a try block?

Reply:
> why do we need to put config.enable_tensor_float_32_xxx to a try block?
Oh my bad ... there's no need for a try / except block. I did this because this API recently changed ... My bad 

> why do we need to save the value and restore it in line 1065?
The configuration might change from test to test and it's environment wide, meaning if we don't reset it would "leak" into the next test

Reply:
The cod hasn't been changed yet?

Reply:
So the try block is indeed needed?

Reply:
try/except block removed.
Logic moved to a contextmanager for readability